config/default.yml

```diff
@@ -43,6 +43,11 @@ RSpec/BeforeAfterAll:
   - spec/support/**/*.rb
   StyleGuide: http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/BeforeAfterAll
 
+RSpec/ContextDescription:
+  Description: "`context` block descriptions should start with 'when'."
+  Enabled: false
```

Comment:
Or should this cop be enabled by default, but turned on in `.rubocop.yml` instead?